dns_check/setup.py

```diff
@@ -1,23 +1,25 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+
+# Always prefer setuptools over distutils
 from setuptools import setup
-from codecs import open  # To use a consistent encoding
+# To use a consistent encoding
+from codecs import open
 from os import path
 
-HERE = path.dirname(path.abspath(__file__))
-
-# Get version info
-ABOUT = {}
-with open(path.join(HERE, 'datadog_checks', 'dns_check', '__about__.py')) as f:
-    exec(f.read(), ABOUT)
+HERE = path.abspath(path.dirname(__file__))
```

Comment:
This needs to remain `os.path.dirname(os.path.abspath(__file__))`

other way around sometimes causes issues https://stackoverflow.com/questions/4934806/how-can-i-find-scripts-directory-with-python

Reply:
hum my bad, I didn't mean to change this actually (blind copy/past ...) - Good catch!